pylutron/__init__.py

```diff
@@ -253,7 +267,7 @@ def _parse_area(self, area_xml):
     area = Area(self._lutron,
                 name=area_xml.get('Name'),
                 integration_id=int(area_xml.get('IntegrationID')),
-                occupancy_group_id=area_xml.get('OccupancyGroupAssignedToID'))
+                occupancy_group=self._occupancy_groups[area_xml.get('OccupancyGroupAssignedToID')])
```

Comment:
for sanity sake, i think we should lookup the group outside here (i.e. just before creating the area in this function), and if it's missing, print a warning and set it to None. Would stink to bug out here if we hit something unexpected.

Reply:
Done